include/extractor/extraction_way.hpp

```diff
@@ -46,6 +46,7 @@ struct ExtractionWay
         roundabout = false;
         circular = false;
         is_startpoint = true;
+        ignore_in_grid = false;
```

Comment:
I did some digging. Seems the profiles are out of date. That flag was removed in https://github.com/Project-OSRM/osrm-backend/commit/faa880d60a162728f8a9dd06c87373b8924d0837